web-console/knikubevirt/WindowsVMs2/WIndowsVMs2.md

```diff
@@ -0,0 +1,15 @@
+# Windows VMs 2
+
+This is following the Windows end to end flow PR and will explain the current constraints in Windows licensing registration that a user needs to be aware of, along with setting up their user profile once the VM has spun up.
+
+At this point, it seems that we can't predetermine the licensing before a VM gets created because no Sys-Prep is available in CNV.
+This means that users will first need to spin up the VM and only then provide the info.
+
```

Comment:
I think you could remove this and just add a title over the next section like `Licensing`